### PR TITLE
OPENEUROPA-1326

### PR DESCRIPTION
**Pull request by dxvargas**
### Description

Minor improvements in README file.

### Change log

- Changed: README.me file.
- Added: github PR template.

### CODE REVIEW

File: github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## OPENEUROPA-[Insert ticket number here]
```
Comment (ademarco) [inline github/PULL_REQUEST_TEMPLATE.md]: Directory and filename are wrong, check docs here: https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/#adding-a-pull-request-template

File: composer.json [hunk 2]
```diff
@@ -34,7 +34,8 @@
             "drupal/core": {
                 "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-07-05/2943172-kernel-test-base-3.patch",
                 "https://www.drupal.org/project/drupal/issues/2599228": "https://www.drupal.org/files/issues/2018-09-25/2599228-74.patch",
-                "https://www.drupal.org/project/drupal/issues/2571475": "https://www.drupal.org/files/issues/outbound_http_requests-2571475-10.patch"
+                "https://www.drupal.org/project/drupal/issues/2571475": "https://www.drupal.org/files/issues/outbound_http_requests-2571475-10.patch",
+                "https://www.drupal.org/project/drupal/issues/2189267": "https://www.drupal.org/files/issues/2018-09-14/2189267-88.patch"
```
Comment (ademarco) [inline composer.json]: This patch is not for development use only, meaning it is needed on production, so it should not be included here.

File: github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dxvargas) [reply github/PULL_REQUEST_TEMPLATE.md]: Ok, changed.

File: composer.json (hunk 2 above)
Comment (dxvargas) [reply composer.json]: Ok, we'll add this patch in a new component, not here.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
-# drupal-core-require
+# drupal-core-require-dev
 
 [![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/drupal-core-require-dev/status.svg?branch=8.6.x)](https://drone.fpfis.eu/openeuropa/drupal-core-require-dev)
 [![Packagist](https://img.shields.io/packagist/v/openeuropa/drupal-core-require-dev.svg)](https://packagist.org/packages/openeuropa/drupal-core-require-dev)
 
-``openeuropa/drupal-core-require-dev`` provides the ``require`` dependencies of ``drupal/core`` as a standalone package. It follows the same release cycle and versioning scheme as Drupal core. You should use the same version constraint for it as you use for Drupal core.
+``openeuropa/drupal-core-require-dev`` provides the ``require_dev`` dependencies of ``drupal/core`` as a standalone package. It follows the same release cycle and versioning scheme as Drupal core. You should use the same version constraint for it as you use for Drupal core.
```
Comment (voidtek) [inline README.md]: ``require_dev`` should be ``require-dev``

**Comment by ademarco** [comment]
We should document all patches in the main README.md.

**Pull request merged**